src/app/core/breadcrumbs/dso-name.service.ts

```diff
@@ -27,7 +27,13 @@ export class DSONameService {
    */
   private readonly factories = {
     Person: (dso: DSpaceObject): string => {
-      return `${dso.firstMetadataValue('person.familyName')}, ${dso.firstMetadataValue('person.givenName')}`;
+      const familyName = dso.firstMetadataValue('person.familyName');
+      const givenName = dso.firstMetadataValue('person.givenName');
+      if (isEmpty(familyName) && isEmpty(givenName)) {
+        return dso.firstMetadataValue('dc.title') || dso.name;
+      } else {
+        return `${familyName}, ${givenName}`;
+      }
```

Comment:
Can we add a test for the new logic to `dso-name.service.spec.ts`?  There's already a test for the old logic, so it just needs to be updated to also test that when one of these name fields is empty, the dc.title is used.

Reply:
done